LoganWilliams_Intro_GitHub/IntroToCodechat/Logan_Williams_Code.rst

```diff
@@ -0,0 +1,41 @@
+.. template documentation master file, created by
```

Comment:
Put all your files in a directory named `lw1551`, instead of `LoganWilliams_Intro_GitHub/IntroToCodechat/`.

Reply:
Also, delete this file.